app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav class="navbar navbar-toggleable-xl navbar-inverse bg-inverse">
+  <div class="navbar-nav">
+    <div class="container-fluid">
+      <ul class="navbar-nav mr-auto">
```

Comment:
what is mr-auto?

Reply:
From 
```
.mr-auto {
  margin-right: auto !important;
}
```
It might be left over from some sample code we looked at.  I'll check again to see if this is actually needed or useful in our case.